frontend/qrencode.lua

```diff
@@ -0,0 +1,1353 @@
+--- The qrcode library is licensed under the 3-clause BSD license (aka "new BSD")
```

Comment:
This file would probably fit better in base, or at least a lib subfolder. Also it looks like CircleCI isn't running on your PR but this file will likely need to be excluded. ^_^

Reply:
I see a lot of bitwise and hex 2 bin ops (without luajit bit module). This is going to be really slow.

Maybe it is better to integrate https://github.com/vincascm/qrencode in base instead?.

Reply:
OK to put it into base, but I don't understand what I have to do for CircleCI.

Reply:
You probably need to go on there and allow it access. The automated tests protect against a lot of issues automatically.

You only need to exclude the file from luacheck if it doesn't pass.